src/index.html

```diff
@@ -4,9 +4,376 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img
+        src="./images/logo.png"
+        alt="LogoMOYO"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="catalog__container">
+        <article class="card catalog__card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+
+          <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```

Comment:
Keep indentation, here and in all other places
```suggestion
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```